OpenSim/Tools/AnalyzeTool.cpp

```diff
@@ -613,6 +613,20 @@ void AnalyzeTool::run(SimTK::State& s, Model &aModel, int iInitial, int iFinal,
     SimTK::Vector stateData;
     stateData.resize(numOpenSimStates);
 
+    const Array<std::string>& stateNames = aStatesStore.getColumnLabels();
+    Array<std::string> modelStateNames = aModel.getStateVariableNames();
+
+    int nsData = stateNames.size() - 1;  //-1 since time is a column
```

Comment:
Can you add a comment saying why this mapping is done? Otherwise, someone else might come here and scrap it, thinking it's unnecessarily complicated.